### ISSUE TITLE
Auto-format sample names

### ISSUE THREAD

**Issue by nick-youngblut** [nf-core/differentialabundance#262]
### Description of feature

The `nf-core/rnaseq` pipeline will generate a `salmon.merged.gene_lengths.tsv` file in which the sample names have been modified if the sample names contain a dash (`-` => `.`).

It would be helpful if the `differentialabundance` pipeline automatically did the same in order to prevent the `DIFFERENTIALABUNDANCE:VALIDATOR` process from failing.

**Comment by pinin4fjords** [comment]
Could you expand on this a little please? Is differentialabundance correcting the sample names in some files and not others, causing the validation to fail?

**Comment by nick-youngblut** [comment]
>  Is differentialabundance correcting the sample names in some files and not others, causing the validation to fail?

Yes, at least the same names in `salmon.merged.gene_lengths.tsv` are modified so that dashes are converted to dots (`'-' => '.'`). It would be helpful if the sample naming was standardized across all output (e.g., add a sample renaming process to the start of the pipeline).
